petcare-enterprise-apps/docs/PETCARE-MOBILE.md

```diff
@@ -1,23 +1,23 @@
 # Pet care sample with Asgardeo Android SDK
 
-## ⚠️ Read this first
+## Prerequisites:
 
-1. Required versions
+Required versions
  ```
  Java version >= 17
  ```
  ```
  Android Studio version >= Giraffe
  ```
 
-#### 1. Setup steps Identity Server
+# 1. Setup steps Identity Server
```

Comment:
Let's write this for Asgardeo

Reply:
Shouldn't we change the title here ?

Reply:
Added with 289d247f19c4b680f81c318068d5a6e2430cfa29